web/src/ProviderEditPage.js

```diff
@@ -191,6 +191,8 @@ class ProviderEditPage extends React.Component {
                   this.updateProviderField("subType", "custom-model");
                 } else if (value === "Azure") {
                   this.updateProviderField("subType", "gpt-4");
+                } else if (value === "Cohere") {
+                  this.updateProviderField("subType", "command");

```

Comment:
Code not added for `this.state.provider.category === "Embedding"`?

Reply:
Embedding for Cohere has existed in "Embedding" condition branch